src/db.js

```diff
@@ -26,7 +26,7 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '1234qwer',
 });
```

Comment:
It's not a good practice to hardcode sensitive data like database password. Even though you've used it as a default value, it might pose a security risk. Consider using a more secure way to handle sensitive data. For example, you could throw an error if the password is not provided in the environment variables.